src/gui/editors/PianoRoll.cpp

```diff
@@ -4153,51 +4071,12 @@ void PianoRoll::setEditMode(int mode)
 
 void PianoRoll::selectAll()
 {
-	if( ! hasValidPattern() )
-	{
-		return;
-	}
-
-	// if first_time = true, we HAVE to set the vars for select
-	bool first_time = true;
-
-	for( const Note *note : m_pattern->notes() )
+	if (hasValidPattern())
```

Comment:
I'd rather keep the return early guard so we don't increase indentation needlessly.
```cpp
if (!hasValidPattern()) { return; }
for (Note* note: m_pattern->notes())
{
  note->setSelected(true);
}
```